docs/langref/src/concepts/layouting.md

```diff
@@ -204,11 +204,12 @@ export component Example inherits Window {
 
 ### Alignment
 
-Each elements is sized according to their `width` or `height` is specified, otherwise it is
+Each elements is sized according to their `width` or `height` if specified, otherwise it is
 set to the minimum size which is set with the min-width or min-height property, or
 the minimum size of an inner layout, whatever is bigger.
-Then, the elements are placed according to the alignment.
-The size of elements is bigger than the minimum size only if the alignment is stretch
+
+Then, the elements are placed according to the alignment. The size of elements
+is bigger than the minimum size only if the alignment is stretch
```

Comment:
"alignment is stretch" doesn't make sense to me (but was there before). @ogoffart Do you remember what is meant here? Should it perhaps be "only if the layout stretches it"?

Reply:
```suggestion
The elements are placed according to the alignment. The size of elements
is bigger than the minimum size only if the `alignment` property of the layout is `LayoutAlignment.stretch` (the default)
```